api/swagger-spec/v1beta3.json

```diff
@@ -12595,6 +12595,10 @@
       "type": "string",
       "description": "human readable message indicating details about why the pod is in this condition"
      },
+     "reason": {
+      "type": "string",
+      "description": "(brief-camelCase) reason indicating details about why the pod is in this condition"
```

Comment:
I'd like reasons to be CamelCase (initial character capitalized), like other API constants.